docgenhtml/plugins/org.polarsys.capella.docgen/templates/pattern._mGFhcAHoEemzNsJkc2kajg/method._mGFhcgHoEemzNsJkc2kajg.pt

```diff
@@ -1 +1 @@
\ No newline at end of file
+<%@ jet package="PACKAGE" class="CLASS" imports="org.eclipse.sirius.viewpoint.DRepresentationDescriptor org.eclipse.sirius.business.api.query.DRepresentationQuery java.util.stream.Collectors org.polarsys.kitalpha.doc.gen.business.core.scope.GenerationGlobalScope org.polarsys.kitalpha.doc.gen.business.core.util.DocGenHtmlUtil org.polarsys.kitalpha.doc.gen.business.core.util.EscapeChars org.eclipse.sirius.diagram.DDiagram org.eclipse.emf.common.util.EList org.eclipse.sirius.viewpoint.DRepresentationDescriptor org.polarsys.kitalpha.doc.gen.business.core.sirius.util.session.DiagramSessionHelper org.eclipse.sirius.business.api.dialect.DialectManager org.eclipse.sirius.viewpoint.description.DAnnotation org.polarsys.capella.core.diagram.helpers.DAnnotationHelper org.polarsys.capella.core.diagram.helpers.naming.DAnnotationSourceConstants org.eclipse.egf.common.helper.* java.util.* org.eclipse.emf.ecore.* org.eclipse.egf.model.pattern.* org.eclipse.egf.pattern.execution.* org.eclipse.egf.pattern.query.* org.eclipse.sirius.diagram.DSemanticDiagram org.polarsys.capella.core.data.capellacore.CapellaElement org.polarsys.capella.core.data.capellacore.NamedElement org.polarsys.capella.docgen.diagram.CapellaHelper org.polarsys.capella.docgen.util.DocGenHtmlCapellaControl org.polarsys.capella.docgen.util.DocGenHtmlCapellaUtil org.polarsys.capella.docgen.util.CapellaDiagramHelper org.polarsys.capella.docgen.util.CapellaServices org.polarsys.capella.docgen.util.CapellaLabelProviderHelper org.polarsys.kitalpha.doc.gen.business.core.preference.helper.DocgenDiagramPreferencesHelper"  %>
```

Comment:
No differences

Reply:
There is a difference: LabelProviderHelper -> CapellaLabelProviderHelper